### DISCUSSION TITLE
jaw movement

### DISCUSSION THREAD

**Discussion by branaway** [pixiv/three-vrm#1100] [Help]
One thing I really like to see is the jaw move with the mouth. Currently jaw does not move along with any mouth expression.   Any way to animation the jaw?

**Comment by 0b5vr** [comment]
There is a community-driven convention called PerfectSync and you might find it helpful:
https://hinzka.hatenablog.com/entry/2021/12/21/222635

Why do you want to move the jaw though?

**Reply by branaway** [reply]
Thanks for the reply. A smile without jaw movement is very unnatural.  The sample model (./models/VRM1_Constraint_Twist_Sample.vrm) has turned out to be jaw bone-less, therefore nothing to manipulate.
